### PR TITLE
corrected grammar (ENG)

### PR DESCRIPTION
**Pull request by appacea**
## What does this PR do ?
Grammar corrections in English

### How should this be manually tested?
Read changes

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "kuzzle-backoffice",
+  "name": "kuzzle admin console",
   "description": "A handy administrative console for Kuzzle.io",
```
Comment (etrousset) [inline bower.json]: Why keep Kuzzle.io here, if we change it later?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "kuzzle-backoffice",
+  "name": "kuzzle-admin-console",
   "version": "2.1.5",
-  "description": "A handy administrative console for Kuzzle.io",
+  "description": "A handy administrative console for Kuzzle",
   "author": "The Kuzzle team <support@kuzzle.io>",
   "private": false,
   "bin": "./bin/kuzzle-backoffice",
```
Comment (etrousset) [inline package.json]: We should not forget to rename the binary also

Comment (appacea) [reply package.json]: yep. didn't want to touch the program part but I can do it if there's no impact

File: bower.json (hunk 1 above)
Comment (appacea) [reply bower.json]: good catch. ill change it.

**Pull request merged**